sample_size/scripts/input_utils.py

```diff
@@ -80,9 +78,43 @@ def get_metric_parameters(parameter_definitions: Dict[str, str]) -> Dict[str, fl
     return parameters
 
 
-def get_metric_metadata() -> Tuple[str, Dict[str, float]]:
-    metric_type = get_metric_type().lower()
-    metric_metadata = get_metric_parameters(METRIC_PARAMETERS[metric_type])
-    metric_metadata["mde"] = get_mde(metric_type)
+def get_variants() -> int:
+    number_of_variants = (
+        input("Enter the number of cohorts for this test \n" "Control + number of treatments: ").strip().lower()
+    )
+    if number_of_variants.isdigit():
+        if int(number_of_variants) < 2:
+            raise ValueError("Error: An experiment must contain at least 2 variants.")
+        return int(number_of_variants)
+    elif number_of_variants == "":
```

Comment:
can you also add `Press Enter to use default variant = 2 if you have only 1 control and 1 treatment`